packages/deployments/contracts/contracts/interfaces/IConnextHandler.sol

```diff
@@ -86,51 +164,128 @@ interface IConnextHandler {
 
   function removeRouter(address router) external;
 
-  function addStableSwapPool(ConnextMessage.TokenId calldata canonical, address stableSwapPool) external;
+  function setMaxRoutersPerTransfer(uint256 _newMaxRouters) external;
 
-  function setupAsset(
-    ConnextMessage.TokenId calldata canonical,
-    address adoptedAssetId,
-    address stableSwapPool
+  function addLiquidityFor(
+    uint256 _amount,
+    address _local,
+    address _router
+  ) external payable;
+
+  function addLiquidity(uint256 _amount, address _local) external payable;
```

Comment:
From an earlier comment : we may want to change this to addRouterLiquidity and addRouterLiquidityFor and removeRouterLiquidity

to avoid confusion between LPing for router stuff from LPing for stableswap